cmd/webserver.go

```diff
@@ -191,14 +189,21 @@ func getInstanceHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func getControlPlaneHandler(w http.ResponseWriter, r *http.Request) {
-	resp := stats.Get()
+	resp := inst.GetControlPlaneStatus()
+	respondWithJSON(w, 200, resp)
+}
+
+func getMetadataHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	id := vars["id"]
+	resp := inst.GetMetadata(id)
```

Comment:
Now here use the newly created method GetActiveActorsCount..mentioned in instances.go file.
md := inst.GetMetadata(id)
resp:= GetActiveActorsCount(md)